src/t8_data/t8_containers.cxx

```diff
@@ -232,46 +232,46 @@ t8_element_t
   return new_elements;
 }
 
-t8_element_t
-  * t8_element_array_index_locidx (t8_element_array_t * element_array,
-                                   t8_locidx_t index)
+t8_element_t       *
+t8_element_array_index_locidx (t8_element_array_t *element_array,
+                               t8_locidx_t index)
 {
   T8_ASSERT (t8_element_array_is_valid (element_array));
   return (t8_element_t *)
     t8_sc_array_index_locidx (&element_array->array, index);
 }
 
-t8_element_t
-  * t8_element_array_index_int (t8_element_array_t * element_array, int index)
+t8_element_t       *
+t8_element_array_index_int (t8_element_array_t *element_array, int index)
 {
   T8_ASSERT (t8_element_array_is_valid (element_array));
   return (t8_element_t *)
     sc_array_index_int (&element_array->array, index);
 }
 
 t8_eclass_scheme_c *
-t8_element_array_get_scheme (t8_element_array_t * element_array)
+t8_element_array_get_scheme (t8_element_array_t *element_array)
 {
   T8_ASSERT (t8_element_array_is_valid (element_array));
   return element_array->scheme;
 }
 
 size_t
-t8_element_array_get_count (t8_element_array_t * element_array)
+t8_element_array_get_count (t8_element_array_t *element_array)
 {
   T8_ASSERT (t8_element_array_is_valid (element_array));
   return element_array->array.elem_count;
 }
 
 size_t
-t8_element_array_get_size (t8_element_array_t * element_array)
+t8_element_array_get_size (t8_element_array_t *element_array)
 {
   T8_ASSERT (t8_element_array_is_valid (element_array));
   return element_array->scheme->t8_element_size ();
 }
 
 t8_element_t       *
```

Comment:
we cannot change the t8_element_t part, since then the check script reports this file as unindented.
We have to leave it and live with it.
Or wrap it in `/* *INDENT-OFF* */ macros, which feels to me as overkill.
But whats your opinion?

Reply:
Then we live with it.